packages/workers.new/tsconfig.json

```diff
@@ -12,6 +11,5 @@
 		"lib": ["es2020"],
 		"noEmit": true
 	},
-	"include": ["src"],
-	"exclude": ["node_modules"]
```

Comment:
could you explain this change?

Reply:
Instead of explicitly including `src`, it should now explicitly _exclude_ `tests`. The functionality should be the same.

Reply:
because node_modules aren't generated within this directory I guess?